### DISCUSSION TITLE
Use scroll wheel to review the log, not command history.

### DISCUSSION THREAD

**Discussion by lunagloaming** [Blightmud/Blightmud#857] [Q&A] [answered]
Just a quick how-do-I... Use scroll wheel to review the log, not command history.

**Comment by LiquidityC** [comment]
You need to enable mouse support for this to work. `/settings` should give you some info.

**Comment by lunagloaming** [comment]
I have typed
```
/settings (displays a list of settings of course)
/settings mouse_enabled (does nothing I can tell)
/settings mouse_enabled on (does nothing I can tell)
```
In all cases, the scroll wheel still only affects command history, not the log.
```
/help settings
```
Is not at all useful, showing no help on how to correctly use the /settings command.  It has a short blurb about the LUA function instead.
```
/help /settingss
```
Shows an error message about no help files.

I think the user friendliness of things could use a little work.  "Discoverability" in Blightmud is wanting.  I also could find no man page regarding it, so no help there either.

**Comment by LiquidityC** [comment, marked as answer]
`/set mouse_enabled on` and then restart Blightmud. There used to be a helpfile explaining this but it seems to have been truncated or gone missing.

EDIT: I dug into the lackluster `/help settings` file and it's now been restored with the information it's intended to contain. If you built your copy of blightmud from source you should be able to update now. In other case you can view the source file here: https://github.com/Blightmud/Blightmud/blob/dev/resources/help/settings.md
